### DISCUSSION TITLE
Interworking the Habitat simulator with the newly captured Matterport 3D environment data

### DISCUSSION THREAD

**Discussion by etri-jeon0123** [facebookresearch/habitat-lab#1437] [Q&A]
Currently, we want to shoot a laboratory using Metaport Pro3 equipment sold by Metaport Company and operate on Habitat.

However, [Habitat - Matterport 3D Research Dataset] provided by Metaport is driven by Habitat as in image (1),
but the experimental space we photographed with Metaport Pro3 equipment does not operate in Habitat as in image (2).
(The code we tested is Habitat-sim's interactive test (python), https://github.com/facebookresearch/habitat-sim) 

**Image(1) Habitat-Matterport 3D Research Dataset**
<img width="901" alt="image" src="https://github.com/facebookresearch/habitat-lab/assets/121843576/ed13a867-63e4-4863-9361-bb60ec347140">

**Image(2) Habitat-Our dataset with Metaport Pro3 equipment**
<img width="899" alt="image" src="https://github.com/facebookresearch/habitat-lab/assets/121843576/fe5bcc42-87f6-4af6-a8e3-007443a1acd8">

The meaning of "not working" is that the .glb file is executed,
but it is impossible to move forward, backward, left, and right, as shown in image (2).

**As far as we're concerned,**
1. Failed to properly generate .nvamesh file provided by Habitat.
2. Pre- and post-processing of 3D.glb data taken is required.

Therefore, the Habitat developer wants an answer on how to solve the above problem.

**Comment by aclegg3** [comment]
@etri-jeon0123 not sure if this question is still fresh, but it looks like you need to re-orient the scene. It is sideways in default orientation. Probably this means it is Z-up rather than Y-up. You can create a [StageAttributes config JSON](https://aihabitat.org/docs/habitat-sim/attributesJSON.html#stageattributes) (see existing datasets as an example) and use the front/up fields to set the orientation correction.
